documentation/modules/exploit/multi/http/processmaker_exec.md

```diff
@@ -0,0 +1,93 @@
+## Description
+
+  This module exploits a PHP code execution vulnerability in the default *neoclassic* skin for [ProcessMaker Open Source](http://www.processmaker.com/) which allows any authenticated user to execute PHP code. The vulnerable skin is installed by default in version 2.x prior to 2.5.2 and cannot be removed via the web interface.
+
+
+## Vulnerable Application
+
+  ProcessMaker is an open source, workflow management software suite, which includes tools to automate your workflow, design forms, create documents, assign roles and users, create routing rules, and map an individual process quickly and easily. It's relatively lightweight and doesn't require any kind of installation on the client computer.
+
+  This module has been tested successfully on ProcessMaker Open Source versions 2.5.1, 2.5.0, 2.0.23 on Windows XP SP3; and version 2.0.45 on Debian Linux.
+
+  The vulnerability was patched in version 2.5.2.
+
+  Source and Installers:
+
+  * [ProcessMaker Open Source 2.0.x](https://sourceforge.net/projects/processmaker/files/ProcessMaker/2.0/)
+  * [ProcessMaker Open Source 2.5.x](https://sourceforge.net/projects/processmaker/files/ProcessMaker/2.5/)
+
+
+## Technical Details
+
+  Several files within the default *neoclassic* skin allow authenticated users to supply a PHP function in the `action` parameter and arguments for the aforementioned function in the `params` parameter. The application calls the supplied function name with the supplied parameters resulting in execution of arbitrary user supplied PHP code.
+
+  The relevant source code for the affected files is shown below.
```

Comment:
While I agree that a module doc might not be the right place for RCA, there's a chance the hard links may be gone in the future. The technical details here are in their own section and don't take up too much space. I see no harm in leaving them alone.

Honestly, we would put technical details in the module description itself, but code snippets would be going to far. Up to you how you want to handle this. I don't mind information overload in documentation so long as the sections are laid out cleanly. Maybe you can move this to an appendix of sorts.